lib/src/main/kotlin/com/wire/integrations/jvm/model/WireMessage.kt

```diff
@@ -399,6 +417,61 @@ sealed interface WireMessage {
         }
     }
 
+    @JvmRecord
+    data class TextEdited @JvmOverloads constructor(
+        override val id: UUID,
+        override val conversationId: QualifiedId,
+        override val sender: QualifiedId? = null,
+        val newContent: String,
+        val newLinkPreviews: List<LinkPreview> = listOf(),
+        val newMentions: List<Mention> = listOf()
+    ) : WireMessage {
+        companion object {
+            /**
+             * Creates a TextEdited message with minimal required parameters.
+             *
+             * Usage from Kotlin:
```

Comment:
We can remove the usage docs after Marian said that in Java the factory are accessible the same as Kotlin

Reply:
removed after latest merge from main with changes on WireMessage